src/scripts/main.js

```
@@ -357,4 +357,44 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
```

Ensure that the '.dashboard' element exists in the DOM before attempting to append rows. If it doesn't exist, the script will fail silently. Consider adding a check to handle this scenario.